main/http_server/axe-os/src/app/services/system.service.ts

```diff
@@ -26,6 +26,7 @@ export class SystemService {
           current: 2237.5,
           fanSpeed: 82,
           temp: 60,
+          tpsTemp: 45,
```

Comment:
i would not have use such a specific naming, a more generic one is prefereable, like vrTemp, so other Voltage Regulator can use this parameter when the TPS is not there...